lib/iris/mesh/components.py

```diff
@@ -2808,6 +2752,31 @@ def location(self):
     def axis(self):
         return self._metadata_manager.axis
 
+    @property
+    def points(self):
+        """The coordinate points values as a NumPy array."""
+        if self.timestamp < self._mesh.timestamp or self.timestamp is None:
+            self._load_points_and_bounds()
+        return self._values
+
+    @points.setter
+    def points(self, value):
+        if value:
+            msg = "Cannot set 'points' on a MeshCoord."
+            raise ValueError(msg)
+
+    @property
+    def bounds(self):
+        if self.timestamp < self._mesh.timestamp or self.timestamp is None:
+            self._load_points_and_bounds()
+        return self.bounds
+
+    @bounds.setter
+    def bounds(self, value):
+        if value:
```

Comment:
Can't use the truthy syntax here because it trips over the ambiguous truth value of array problem.